log/logging.go

```diff
@@ -135,19 +134,14 @@ func SetPrefix(prefix string) {
 // Fatal outputs a severe error message just before terminating the process.
 // Use judiciously.
 func Fatal(id, description string, keysAndValues ...interface{}) {
-	if Level < LevelFatal {
-		return
-	}
-	DefaultLogger.logMessage(LevelFatalName, id, description, keysAndValues...)
-	osExit(1)
+	keysAndValues = append([]interface{}{"id", id}, keysAndValues...)
```

Comment:
It actually looks for this in the formatter and removes it. The end result is that all logging remains the same, unless someone logged like `log.Debug("An ID", "description", "id", "will replace 'An ID'")` But.. meh?

Also your suggestion would mess up the json formatter's description.